debianutils.yaml

```diff
@@ -0,0 +1,202 @@
+package:
+  name: debianutils
+  version: 5.23.2
+  epoch: 0
+  description: Miscellaneous utilities specific to Debian
+  copyright:
+    - license: GPL-2.0-or-later
+    - license: SMAIL-GPL
+  dependencies:
+    replaces:
+      - busybox
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - coreutils
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://salsa.debian.org/debian/debianutils.git
+      tag: debian/${{package.version}}
+      expected-commit: b999bb0969471d302f1db85e58f17dcb64f274c2
+
+  - uses: autoconf/configure
+
+  - runs: |
+      # Build only the C programs, skip po4a translations
+      make run-parts tempfile ischroot
+
+  - runs: |
+      # Install manually to avoid make install tripping on .po generation
+      install -d ${{targets.destdir}}/usr/bin
+      install -m755 run-parts ${{targets.destdir}}/usr/bin/
+      install -m755 tempfile ${{targets.destdir}}/usr/bin/
+      install -m755 ischroot ${{targets.destdir}}/usr/bin/
+      install -m755 which.debianutils ${{targets.destdir}}/usr/bin/which
+      install -m755 savelog ${{targets.destdir}}/usr/bin/
+      install -m755 installkernel ${{targets.destdir}}/usr/bin/
+      install -m755 add-shell ${{targets.destdir}}/usr/bin/
+      install -m755 remove-shell ${{targets.destdir}}/usr/bin/
+      install -m755 update-shells ${{targets.destdir}}/usr/bin/
+      # Copy license info in to cover which.debianutils and savelog
+      install -d ${{targets.destdir}}/usr/share/doc/debianutils
+      install -m644 debian/copyright ${{targets.destdir}}/usr/share/doc/debianutils/
+
+  - uses: strip
+
+test:
+  environment:
+    contents:
+      packages:
+        - coreutils
+  pipeline:
+    # TODO: add ischroot test back once arm64 CI is on qemu
+    - runs: |
+        set -euo pipefail
+
+        # Test run-parts
+        run-parts --version
+
+        # Test tempfile
+        tempfile --version
+
+        # Test ischroot
+        ischroot --version
+
+        # Test which.debianutils (no --version flag)
+        which.debianutils echo
+
+        # Test shell management scripts exist
+        test -f /usr/bin/add-shell
+        test -f /usr/bin/remove-shell
+        test -f /usr/bin/update-shells
+        test -f /usr/bin/installkernel
+        test -f /usr/bin/savelog
```

Comment:
If we want to check for existence of a file, I'd prefer we used `stat`. We get more verbose information about the file we're looking for versus only whether it exists or not. If debug isn't enabled, we also lose visibility when `test` fails so the problem because less obvious when we're debugging failing tests